node21-expressjs/README.md

```diff
@@ -0,0 +1,16 @@
+# Node 21 ExpressJS Example
+
+This example uses the [ExpressJS Node framework](https://expressjs.com/).
+To run this example, first [install the `kraft` CLI tool](https://unikraft.org/docs/cli).
+
+Then, clone this repository and `cd` into this directory.
+To deploy this application on KraftCloud, invoke:
+
+```
+kraft cloud deploy -p 443:8080 -M 256 .
```

Comment:
Is `256` enough? for the test `2048` is used

Reply:
actually `512` is used in the test

Reply:
Yes, `256` is enough. I updated the [workflow PR](https://github.com/kraftcloud/catalog/pull/128) to use `256`.